run-tests-with-build-drools-parent/drools-project-sources-test/pom.xml

```diff
@@ -46,17 +46,18 @@
           </execution>
         </executions>
       </plugin>
-
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-invoker-plugin</artifactId>
         <configuration>
-          <parallelThreads>1</parallelThreads>
-          <!--
-          script deciding if included project is:
-            executed (either script missing or returning true)
-              or
-            skipped (when returns false or throws error) -->
+          <goals>
+            <goal>clean</goal>
+            <goal>org.kie:unpack-build-maven-plugin:1.0-SNAPSHOT:unpack-build</goal>
```

Comment:
can we use the property we have for the version? (Noticed only after merging in 1.24.x).

Reply:
Sure!